blast/blast_data_gen

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/perl
+
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "Usage: blast_data_gen number_contigs length_of_contigs\n";
```

Comment:
I looked into it. It depends on the version of Perl, but in the more recent versions will automatically seed the random number generator (with what, I am not quite sure). Before 5.004 they seeded it with time, but that is no longer used. That said, I can add it but I am not sure it will improve randomness.